.github/workflows/call-build-windows.yaml

```diff
@@ -45,14 +45,17 @@ jobs:
         config:
           - name: "Windows 32bit"
             arch: x86
-            vcpkg_triplet: x86-windows-static
-            openssl_root_dir_opt: -DOPENSSL_ROOT_DIR=C:\vcpkg\packages\openssl_x86-windows-static
+            openssl_dir: C:\Program Files (x86)\OpenSSL-Win32
+            chocolatey_opt: --x86
           - name: "Windows 64bit"
             arch: x64
-            vcpkg_triplet: x64-windows-static
-            openssl_root_dir_opt: ""
+            openssl_dir: C:\Program Files\OpenSSL-Win64
+            chocolatey_opt: --x64
     permissions:
       contents: read
+    # Default environment variables can be overridden below. To prevent library pollution.
```

Comment:
```suggestion
    # Default environment variables can be overridden below. To prevent library pollution - without this other random libraries may be found on the path leading to failures.
```

Reply:
Ah, great!